specification/servicefabric/resource-manager/Microsoft.ServiceFabric/preview/2017-07-01-preview/application.json

```diff
@@ -2124,7 +2097,7 @@
       "description": "The customer subscription identifier.",
       "required": true,
       "type": "string",
-      "x-ms-parameter-location": "method"
+      "x-ms-parameter-location": "client"

```

Comment:
Are you sure about this change? This will be breaking change in the SDK. So, Please confirm if you are absolutely sure that this is what you want

Reply:
These resources were not part of the SDK until now, so this is not a breaking change.